src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,59 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.HashMap;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
-
+        String line;
```

Comment:
Better move this variable inside `try` block

Reply:
i did better